validate/isimagename.go

```diff
@@ -13,6 +13,11 @@ const alphabet = letters + digits + separators
 func isImageName(fl validator.FieldLevel) bool {
 	name := fl.Field().String()
 
+	// default value
+	if name == "" {
```

Comment:
CustomBuilderImage is by default empty.
Allow it as a value, as we have a proper fallback in any case.